dotnet/src/Connectors/Connectors.Google/Core/Gemini/Clients/GeminiChatCompletionClient.cs

```diff
@@ -161,11 +162,29 @@ internal sealed class GeminiChatCompletionClient : ClientBase
 
         for (state.Iteration = 1; ; state.Iteration++)
         {
-            var geminiResponse = await this.SendRequestAndReturnValidGeminiResponseAsync(
-                    this._chatGenerationEndpoint, state.GeminiRequest, cancellationToken)
-                .ConfigureAwait(false);
+            GeminiResponse geminiResponse;
+            List<GeminiChatMessageContent> chatResponses;
+            using (var activity = ModelDiagnostics.StartCompletionActivity(
+                this._chatGenerationEndpoint, this._modelId, "Google", chatHistory, executionSettings))
```

Comment:
Does  `"Google"` constant go to `gen_ai.system`? It (and all the others) should be lowercase then. Language-specific preferences does not apply and we need to have common casing everywhere. Lowercase is the existing approach for otel